epub33/rs/index.html

```diff
@@ -361,7 +361,7 @@ <h3>Network Access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External Links</h3>
 
-				<p>Reading Systems SHOULD open links that resolve outside the <a>EPUB Publication</a> in a new browser
+				<p data-tests="pub-external-links">Reading Systems SHOULD open links that resolve outside the <a>EPUB Publication</a> in a new browser
```

Comment:
Alas!, due to the quirks in the way the test environment works this must be a fragment ID. To save time, I will merge this right away.


```suggestion
				<p data-tests="#pub-external-links">Reading Systems SHOULD open links that resolve outside the <a>EPUB Publication</a> in a new browser
```